storage/engine/mvcc_test.go

```diff
@@ -1729,13 +1726,7 @@ func TestMVCCStatsBasic(t *testing.T) {
 // underlying engine.
 func TestMVCCStatsWithRandomRuns(t *testing.T) {
 	defer leaktest.AfterTest(t)
-	var seed int64
-	err := binary.Read(crypto_rand.Reader, binary.LittleEndian, &seed)
-	if err != nil {
-		t.Fatalf("could not read from crypto/rand: %s", err)
-	}
-	log.Infof("using pseudo random number generator with seed %d", seed)
-	rng := rand.New(rand.NewSource(seed))
+	rng := util.NewPseudoRand()
```

Comment:
The point here was to print out the seed so that the test can be restarted deterministically to track down failures. Could you change this code to have NewPseudoRand return two values: the Rand object and also the seed used? Then print out the seed here as before using log.Infof().